marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/customStep/getCustomStep.sjs

```diff
@@ -17,17 +17,19 @@
 
 xdmp.securityAssert("http://marklogic.com/data-hub/privileges/read-custom", "execute");
 
-const Artifacts = require('/data-hub/5/artifacts/core.sjs')
+const Artifacts = require('/data-hub/5/artifacts/core.sjs');
 var stepName;
 
 let dataHubOptions = ["sourceQuery", "sourceQueryLimit", "sourceQueryIsScript", "constrainSourceQueryToJob", "provenanceGranularityLevel", "acceptsBatch",
-  "stepUpdate", "collections" , "additionalCollections", "permissions", "outputFormat", "sourceDatabase", "targetDatabase", "targetEntityType"];
+  "stepUpdate", "collections" , "additionalCollections", "permissions", "outputFormat", "sourceDatabase", "targetDatabase", "targetEntityType", "mapping",
```

Comment:
I don't think we expect "mapping" or "validateEntity" here, as those are specific to legacy and new mapping steps. Why are they listed?